src/python/tests/gff3/test_simplifier.py

```diff
@@ -0,0 +1,365 @@
+        param(
+            "transposable_element",
+            "transposon",
+            None,
+            "transposable_element",
+            "transposon",
+            does_not_raise(),
+            id="MGE, transposon",
+        ),
+        param(
+            "transposable_element",
+            "transposon:LOREM",
+            None,
+            "transposable_element",
+            "transposon (LOREM)",
+            does_not_raise(),
+            id="MGE, transposon named",
+        ),
+        param(
+            "transposable_element",
+            "retrotransposon:LOREM",
+            None,
+            "transposable_element",
+            "retrotransposon (LOREM)",
+            does_not_raise(),
+            id="MGE, retrotransposon named",
+        ),
+        param(
+            "transposable_element",
+            "UNKNOWNtransposon:LOREM",
+            None,
+            "transposable_element",
+            None,
+            raises(GFFParserError),
+            id="MGE, unknown type",
+        ),
+        param(
+            "transposable_element",
+            "transposon",
+            "PROD",
+            "transposable_element",
+            "PROD",
+            does_not_raise(),
+            id="MGE, transposon, product exists",
+        ),
+    ],
+)
+def test_normalize_non_gene(
+    in_type: str,
+    in_mobile_type: Optional[str],
+    in_product: Optional[str],
+    out_type: str,
+    out_description: Optional[str],
+    expectation: ContextManager,
+) -> None:
+    """Test gene create gene for lone CDS."""
+    simp = GFFSimplifier()
+    feat = SeqFeature(None, in_type)
+    feat.qualifiers = {"source": "LOREM"}
+    if in_mobile_type is not None:
+        feat.qualifiers["mobile_element_type"] = [in_mobile_type]
+    if in_product is not None:
+        feat.qualifiers["product"] = [in_product]
+    feat.sub_features = []
+    with expectation:
+        new_feat = simp.normalize_non_gene(feat)
+        if new_feat is not None:
+            assert new_feat.type == out_type
+            if out_description is not None:
+                assert new_feat.qualifiers == feat.qualifiers
+
+
+@pytest.mark.parametrize(
+    "in_type, name, out_type, expectation",
+    [
+        param("mRNA", "", "mRNA", does_not_raise(), id="mRNA no change"),
+        param("C_gene_segment", "", "C_gene_segment", raises(GFFParserError), id="no standard name"),
+        param("C_gene_segment", "immunoglobulin", "IG_C_gene", does_not_raise(), id="C immunoglobulin"),
+        param("C_gene_segment", "ig", "IG_C_gene", does_not_raise(), id="C ig"),
+        param("V_gene_segment", "t-cell", "TR_V_gene", does_not_raise(), id="V t-cell"),
+        param("V_gene_segment", "T_cell", "TR_V_gene", does_not_raise(), id="V T_cell"),
+        param("V_gene_segment", "Lorem Ipsum", "", raises(GFFParserError), id="V T_cell"),
+    ],
+)
+def test_format_gene_segments(
+    in_type: str,
+    name: Optional[List[str]],
```

Comment:
```suggestion
    name: str,
```
This change may allow you to put all the arguments in a single line.